usl_pipeline/usl_pipeline/writers/elevation_writers.py

```diff
@@ -0,0 +1,26 @@
+import typing
+
+from usl_pipeline.shared.geo_data import Elevation
+
+
+def write_to_esri_ascii_raster_file(elevation: Elevation, file: typing.TextIO) -> None:
+    """Writes elevation data to a text file in Esri ASCII format.
+
+    Args:
+        elevation: Elevation data.
+        file: Output file/stream to write data to.
+    """
+    elv_header = elevation.header
+    file.write("ncols {}\n".format(elv_header.col_count))
+    file.write("nrows {}\n".format(elv_header.row_count))
+    file.write("xllcorner {}\n".format(elv_header.x_ll_corner))
+    file.write("yllcorner {}\n".format(elv_header.y_ll_corner))
+    file.write("cellsize {}\n".format(elv_header.cell_size))
+    file.write("NODATA_value {}\n".format(elv_header.nodata_value))
+
+    if elevation.data is not None:
```

Comment:
Might raise an exception or make some other indication that there's nothing to write -- I'd imagine attempting to write empty elevation data means a user error?

Reply:
Done